src/Upgrade/Infrastructure/VersionControlSystem/Generator/PullRequestDataGenerator.php

```diff
@@ -89,19 +89,25 @@ public function buildBody(
     protected function buildHeaderText(StepsResponseDto $stepsResponseDto, ?int $releaseGroupId): string
     {
         $releaseGroupStatDto = $stepsResponseDto->getReleaseGroupStatDto();
-
-        $text = sprintf('Upgrader installed %s release group(s) ', $releaseGroupStatDto->getAppliedRGsAmount());
+        $composerDiffDto = $stepsResponseDto->getComposerLockDiff();
```

Comment:
I'm not sure that you can use getComposerLockDiff for getting some statistics information because it's overrided after each release group iteration in sequential strategy

Reply:
Check bug. the issue is when $stepsResponseDto->getComposerLockDiff() has changed packages we have 0 RPs